src/index.ts

```diff
@@ -722,6 +728,7 @@ class Storyblok {
 
 	public async flushCache(): Promise<this> {
 		await this.cacheProvider().flush()
+		this.clearCacheVersion();
```

Comment:
I think you could remove the `;`
```suggestion
		this.clearCacheVersion()
```